guides/node-operators/run-a-super-world-computer-node.md

```diff
@@ -2,89 +2,121 @@
 
 This guide will help you get a Super World Computer node up and running.
 
-## Beta Testnet
+## Software dependencies
 
+| Dependency                                                    | Version  | Version Check Command |
+| ------------------------------------------------------------- | -------- | --------------------- |
+| [git](https://git-scm.com/)                                   | `^2`     | `git --version`       |
+| [go](https://go.dev/)                                         | `^1.21`  | `go version`          |
+| [make](https://linux.die.net/man/1/make)                      | `^3`     | `make --version`      |
 
-### Steps
+## Beta Testnet
 
-1. Prepare [rollup.json](https://github.com/ethstorage/pm/blob/main/L2/assets/beta_testnet_rollup.json) and [genesis.json](https://github.com/ethstorage/pm/blob/main/L2/assets/beta_testnet_genesis.json).
+### Steps
 
+1. Follow the following steps to build a node. The steps are basically the same as in [Optimism's documentation](https://docs.optimism.io/builders/node-operators/tutorials/node-from-source), the only difference is that here we use the `beta_testnet` branch of both [our optimism fork](https://github.com/ethstorage/optimism/tree/beta_testnet) and [our op-geth fork](https://github.com/ethstorage/op-geth/tree/beta_testnet) instead.
 
-2. Follow the steps [here](https://docs.optimism.io/builders/chain-operators/tutorials/create-l2-rollup) to build `op-node` and `op-geth`, the only difference is: use the `beta_testnet` branch of both [optimism](https://github.com/ethstorage/optimism/tree/beta_testnet) and [op-geth](https://github.com/ethstorage/op-geth/tree/beta_testnet) instead.
+- 1.1 Build `op-geth`
+    ```bash
+    git clone -b beta_testnet https://github.com/ethstorage/op-geth.git
+    cd op-geth && make geth
```

Comment:
I'd prefer `pushd xxx && make yyy && popd` which automatically recovers the cwd.